packages/backend/src/schedulerMetrics.ts

```diff
@@ -0,0 +1,25 @@
+import opentelemetry, { ValueType } from '@opentelemetry/api';
```

Comment:
see docs for adding metrics https://opentelemetry.io/docs/instrumentation/js/manual/#metrics